ddpui/tests/core/dashboard_chat/test_vector_store.py

```
@@ -0,0 +1,310 @@
+            "documents": [["matched content"]],
+            "metadatas": [[{"source_type": "org_context"}]],
+            "distances": [[0.12]],
+        }
+
+    def upsert(self, **kwargs):
+        self.upsert_calls.append(kwargs)
+        for document_id, document, metadata in zip(
+            kwargs["ids"],
+            kwargs["documents"],
+            kwargs["metadatas"],
+        ):
```

Ruff (0.15.6)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)